### PR TITLE
Yeksi Naa Reports | Fix Dashboard 1

### PR DESCRIPTION
**Pull request by dmydlo**
Hi @emord,
These changes are combining necessary changes for dashboard 1 as well as provide precise averages.
Changes were necessary due to use of: [{}] * len(x)

### CODE REVIEW

File: custom/yeksi_naa_reports/sqldata.py [hunk 1]
```diff
@@ -190,49 +190,55 @@ class AvailabilityData(VisiteDeLOperateurDataSource):
 
     def calculate_total_row(self, rows):
         total_row = ['Availability (%)']
+        total_nominator = 0
```
Comment (emord) [inline custom/yeksi_naa_reports/sqldata.py]: Can you change this to `total_numerator`

File: custom/yeksi_naa_reports/sqldata.py [hunk 2]
```diff
@@ -190,49 +190,55 @@ class AvailabilityData(VisiteDeLOperateurDataSource):
...
+        total_denominator = 0
         if self.loc_id == 'pps_id':
             data = {}
             for i in range(len(self.months)):
                 data[i] = {
-                    'pps_is_outstock': 0,
-                    'pps_count': 0
+                    'pps_is_available': sum(
+                        [pps_data[i + 1] for pps_data in rows if pps_data[i + 1] != 'no data entered']
```
Comment (emord) [inline custom/yeksi_naa_reports/sqldata.py]: Does this work if you omit the outermost `[]` ? That would create a generator expression and avoid creating an extra list

**Review by emord — APPROVED** [review]
This code's a bit hard to follow. Would it be easier to create a `Model` or use `namedtuple` to have more semantic variable names

File: custom/yeksi_naa_reports/sqldata.py (hunk 1 above)
Comment (dmydlo) [reply custom/yeksi_naa_reports/sqldata.py]: Sure, good catch.
